python/python/lance/tf/data.py

```diff
@@ -215,7 +215,7 @@ def gen_fragments(fragments):
                 ):
                     yield LanceFragment(dataset, int(f))
                 elif isinstance(f, FragmentMetadata):
-                    yield LanceFragment(dataset, f.fragment_id)
+                    yield LanceFragment(dataset, f.id)
```

Comment:
I changed `fragment_id` to `id`. because fragment metadata only has attribute `id`